### ISSUE TITLE
ILBE Extend Antenna function

### ISSUE THREAD

**Issue by Saborknight** [2bnb/2bnb-extras#45]
**Arma 3 Version:** `1.90` (stable)
**2BNB Extras Version:** `1.9.1` (stable)
**2BNB Modpack Version:** `1.0.5`

**Description:**
ILBE Backpack antenna does not extend, instead it gives an error message

**Steps to reproduce:**
- Equip an ILBE backpack
- ACE self-interact >> Equipment >> Extend Antenna

**Where did the issue occur?**
- 2BNB Server

**RPT log file:**
```
14:52:07 Error in expression <e _result = [];

_result pushBackUnique _overrideLR; 
if (_activeLR isEqualTo _v>
14:52:07   Error position: <_overrideLR; 
if (_activeLR isEqualTo _v>
14:52:07   Error Undefined variable in expression: _overridelr
14:52:07 File z\tfar\addons\core\functions\fnc_lrRadiosList.sqf, line 30
```

**Screenshots/Video**
![20190412145213_1](https://user-images.githubusercontent.com/9272259/56019305-97a8e400-5d47-11e9-90fc-b827ad42e432.jpg)

**Comment by UrekD** [comment]
Should be fixed in dc871d52a5712479a988041e0e038445b4f3e08f, but needs to be tested on the server!
*The fix it only temporary everything else is on Dedmen becouse one of the fuctions called (TFAR function) has a variable that is for some reason used but never defined?

**Comment by UrekD** [comment]
Fixed in latest TFAR- Remove the patch made for it in extras!
